package.json

```diff
@@ -5,19 +5,26 @@
   "main": "index.js",
   "scripts": {
     "start": "npx tsc && cp -R views dist && node dist/app.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "mocha -require ts-node/register test/unit/**/*.ts",
+    "test-ui": "mocha -require ts-node/register --timeout 10000 test/ui/**/*.ts"
   },
   "author": "",
   "license": "ISC",
   "dependencies": {
     "@types/express-session": "^1.17.7",
     "axios": "^1.5.0",
+    "axios-mock-adapter": "^1.22.0",
+    "chai": "^4.3.8",
+    "dotenv": "^16.3.1",
```

Comment:
You can use the environment variables without the `dotenv` dependency. If you remove it you can use the environment variables as you have already used them. If adding environment variables could you also add them to a list in the `README.md` file?